omnigibson/envs/env_base.py

```diff
@@ -280,6 +268,33 @@ def _load_objects(self):
 
         assert og.sim.is_stopped(), "Simulator must be stopped after loading objects!"
 
+    def _load_external_sensors(self):
+        """
+        Load any additional custom external sensors into the scene
+        """
+        assert og.sim.is_stopped(), "Simulator must be stopped before loading external sensors!"
+        sensors_config = self.env_config["external_sensors"]
+        if sensors_config is not None:
+            self._external_sensors = dict()
+            for i, sensor_config in enumerate(sensors_config):
+                # Add a name for the object if necessary
+                if "name" not in sensor_config:
+                    sensor_config["name"] = f"external_sensor{i}"
+                # Determine prim path if not specified
+                if "prim_path" not in sensor_config:
+                    sensor_config["prim_path"] = f"/World/{sensor_config['name']}"
+                # Pop the desired position and orientation
+                local_position, local_orientation = sensor_config.pop("local_position", None), sensor_config.pop("local_orientation", None)
```

Comment:
Should we assert local position and local orientation are not None? Or is it okay for them to be None?

Reply:
hmmm, good point. Originally i had expected them to be optional, similar to all the other objects / robots being specified. What do you think?

Reply:
keeping None for now